catalyst-gateway/bin/src/db/index/schema/mod.rs

```diff
@@ -16,8 +16,11 @@ use crate::{settings::cassandra_db, utils::blake2b_hash::generate_uuid_string_fr
 /// This constant is ONLY used by Unit tests to identify when the schema version will
 /// change accidentally, and is NOT to be used directly to set the schema version of the
 /// table namespaces.
-#[allow(dead_code)]
-const SCHEMA_VERSION: &str = "08193dfe-698a-8177-bdf8-20c5691a06e7";
+pub const SCHEMA_VERSION: &str = "08193dfe-698a-8177-bdf8-20c5691a06e7";
```

Comment:
This MUST never be public, and MUST never be used anywhere outside unit tests, as dictated by the comment.